ansible/deploy-dev.yml

```diff
@@ -0,0 +1,9 @@
+- name: Deploy Playbook
+  hosts: all
+  tasks:
+    - name: Deploy html
+      copy:
+        src: ../site/public/
+        dest: /var/www/dev/
```

Comment:
the `ansible/deploy-dev.yml` and `ansible/deploy-staing.yml` are almost the same. is there any reason you didn't use a template for them?

Reply:
My previous thought was that this is too short and simple so that no need to use a template... But right, template will make it more elegant.